expertise/models/specter2_scincl/scincl.py

```diff
@@ -190,10 +194,11 @@ def embed_publications(self, publications_path=None):
             f.writelines(pub_jsonl)
 
     def all_scores(self, publications_path=None, submissions_path=None, scores_path=None, p2p_path=None):
-        def load_emb_file(emb_file):
+        def load_emb_file(emb_file, load_weight=False):
```

Comment:
Why is `load_weight` here False by default? What does it mean? Why isn't it always True, for example?

Reply:
The model shouldn't load a weight matrix if there is no `weight` field in the JSON objects of the embedding file. There are no weights for submissions and there are no weights for publications if there is no `weight_specification` field:
`load_weight=self.venue_specific_weights`

Reply:
if there is no weight_specification, aren't all the submissions and publications having weight=1? or having a weight matrix with all 1 is too expensive?

Reply:
If there's no `weight_specification`, there are no `weight` keys and no extra computation is done at all to preserve the original workflow